### PR TITLE
Add tests to the external modules feature.

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: cmake/iotjs.cmake [hunk 1]
```diff
@@ -48,7 +48,7 @@ function(getListOfVarsStartWith prefix varResult)
     string(REPLACE "." "\\." prefix ${prefix})
     foreach(var ${vars})
       string(REGEX MATCH
-             "(^|;)${prefix}([A-Za-z0-9_]+[A-Za-z]+)[A-Za-z0-9_.]*"
+             "(^|;)${prefix}([A-Za-z0-9_]+[A-Za-z0-9]+)[A-Za-z0-9_.]*"
```
Comment (LaszloLango) [inline cmake/iotjs.cmake]: @bzsolt please check this fix if you have some time.

**Comment by daeyeon** [comment]
(Informal) LGTM

**Comment by yichoi** [comment]
LGTM after @bzsolt confirmed

**Comment by bzsolt** [comment]
@yichoi Looks good to me.

**Pull request merged**